src/main/scala/xiangshan/backend/issue/Scheduler.scala

```diff
@@ -119,6 +120,11 @@ abstract class SchedulerImpBase(wrapper: Scheduler)(implicit params: SchdBlockPa
   // Modules
   val dispatch2Iq: Dispatch2IqImp = wrapper.dispatch2Iq.module
   val issueQueues: Seq[IssueQueueImp] = wrapper.issueQueue.map(_.module)
+  if (params.isIntSchd) {
+    dispatch2Iq.io.IQValidNumVec.get := io.IQValidNumVec
+    io.IQValidNumVec := VecInit(issueQueues.take(4).map(_.io.validCntDeqVec))
```

Comment:
PLZ explain how to get these immediate numbers and name them.

Reply:
MixedVec